frontend/public/components/filter-toolbar.tsx

```diff
@@ -98,11 +99,30 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
     labelFilter = filterTypeMap[FilterType.LABEL],
   } = props;
 
+  const { t } = useTranslation();
+  const translateFilterType = (value: string) => {
+    switch (value) {
+      case 'Name':
+        return t('filter-toolbar~Name');
+      case 'Label':
+        return t('filter-toolbar~Label');
+      case 'labels':
+        return t('filter-toolbar~labels');
+      case 'name':
+        return t('filter-toolbar~name');
```

Comment:
I'm a little confused why we need both `Name` and `name` and `Label` and `label`. Shouldn't it always be one or the other?